mx.c

```diff
@@ -79,6 +79,10 @@
 #ifdef ENABLE_NLS
 #include <libintl.h>
 #endif
+#ifdef __APPLE__
+# include <string.h>
```

Comment:
why string?

Reply:
I used to have same problems in older version on OSX. So i included both to be safe
As of now, I tested it on 10.12. It is working with only <xlocale.h> included.
Shall i remove <string.h> then?
Thanks.